.travis.yml

```diff
@@ -10,9 +10,6 @@ jobs:
       env: TOXENV=pylint
     - python: "3.8"
       env: TOXENV=pydocstyle
-    - python: "3.5"
-      env: TOXENV=py35
-      after_success: coveralls
```

Comment:
I think all changes currently are still 3.5 compatible. Maybe we should remove 3.5 in an extra PR?